x-pack/filebeat/module/palo_alto/log/config/input.yml

```diff
@@ -0,0 +1,156 @@
+{{ if eq .input "syslog" }}
+
+type: syslog
+protocol.udp:
+  host: "{{.syslog_host}}:{{.syslog_port}}"
+
+{{ else if eq .input "file" }}
+
+type: log
+paths:
+{{ range $i, $path := .paths }}
+  - {{$path}}
+{{ end }}
+exclude_files: [".gz$"]
+
+{{ end }}
+
+tags: {{.tags}}
+
+processors:
+{{ if .convert_timezone }}
+  - add_locale: ~
+{{ end }}
+
+  - decode_csv_fields:
+      fields:
+        message: csv
+
+  - extract_array:
+      field: csv
+      overwrite_keys: true
+      mappings:
+        event.created: 1
+        observer.serial_number: 2
+        _temp_.message_type: 3
+        _temp_.message_subtype: 4
+        _temp_.generated_time: 6
+
+  - extract_array:
+      when:
+        equals:
+          _temp_.message_type: TRAFFIC
+      field: csv
+      overwrite_keys: true
+      omit_empty: true
+      fail_on_error: false
+      mappings:
+        client.ip: 7
+        source.ip: 7
```

Comment:
Let's copy this yet another time to `source.address`. Same for destination.address.